extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1195,12 +1195,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         });
   }
 
+  /** @override */
   getPreconnectUrls() {
-    const urls = ['https://partner.googleadservices.com'];
     if (this.preloadSafeframe_) {
-      urls.push(SAFEFRAME_ORIGIN);
+      this.preconnect.preload(this.getSafeframePath());
```

Comment:
Why do we have both `getSafeframePath_` and `SAFEFRAME_ORIGIN`?

Reply:
getSafeframePath builds the full URL while SAFEFRAME_ORIGIN just returns the domain/origin.  Updated to that SAFEFRAME_ORIGIN within safeframe-host.js is used